build.gradle

```diff
@@ -0,0 +1,71 @@
+ext {
+    aggdesigner_algorithm_version = "6.0"
+    airline_version = "2.7.0"
+    avatica_core_version = "1.16.0-POLYPHENYDB"
+    avatica_server_version = "1.16.0-POLYPHENYDB"
+    byteunits_version = "0.9.1"
+    calcite_linq4j_version = "1.18.0"
+    cassandra_all_version = "3.11.2"
+    cassandra_driver_core_version = "3.6.0"
+    cassandra_unit_version = "3.5.0.1"
+    commons_codec_version = "1.12"
+    commons_compiler_version = "3.0.11"
+    commons_dbcp2_version = "2.5.0"
+    commons_io_version = "2.4"
+    commons_lang3_version = "3.8" // alternative version: 3.2
+    commons_pool2_version = "2.6.0"
+    commons_text_version = "1.7"
+    config_version = "1.2.1"
+    elasticsearch_rest_client_version = "6.2.4"
+    elasticsearch_version = "6.2.4"
+    esri_geometry_api_version = "2.2.0"
+    fongo_version = "2.1.1"
+    foodmart_data_hsqldb_version = "0.3"
+    foodmart_data_json_version = "0.4"
+    foodmart_queries_version = "0.4.1"
+    geode_core_version = "1.6.0"
+    gson_version = "2.8.5"
+    guava_version = "19.0" // alternative version: 11.0.2
+    h2_version = "1.4.197"
+    hadoop_client_version = "2.7.5"
+    hadoop_common_version = "2.7.5"
+    hamcrest_core_version = "1.3"
+    hsqldb_version = "2.5.0" // alternative version: 2.4.1
+    httpclient_version = "4.5.6"
+    httpcore_version = "4.4.10"
+    jackson_annotations_version = "2.9.6"
+    jackson_core_version = "2.9.6"
+    jackson_databind_version = "2.9.6"
+    jackson_dataformat_yaml_version = "2.9.6"
+    janino_version = "3.0.11"
+    java_diff_version = "1.1.2"
+    javacc_version = "4.0"
+    joda_time_version = "2.8.1"
+    json_path_version = "2.4.0"
+    jsoup_version = "1.11.3"
+    jsr305_version = "3.0.1"
+    junit_version = "4.12"
+    lang_painless_version = "6.2.4"
+    log4j_api_version = "2.11.2" // alternative version: 2.11.0
+    log4j_core_version = "2.11.2" // alternative version: 2.11.0
+    log4j_slf4j_impl_version = "2.11.2"
+    mockito_core_version = "2.22.0"
+    mongo_java_driver_version = "3.5.0"
+    mysql_connector_java_version = "5.1.20"
+    natty_version = "0.13"
+    opencsv_version = "2.3"
+    oshi_core_version = "4.0.0" // alternative version: 3.13.2
```

Comment:
Please use version 3.13.2 instead.

Reply:
Trying that out right now, but first testing what happens if the `dbms` subproject is downgraded from `4.0.0` to `3.13.2`